tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_section_size.py

```diff
@@ -0,0 +1,97 @@
+"""
+EOF Container, test custom_size field for sections
+"""
+
+from enum import Enum
+
+import pytest
+
+from ethereum_test_tools import EOFTestFiller
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools.eof.v1 import Container, EOFException, Section, SectionKind
+
+from .. import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-3540.md"
+REFERENCE_SPEC_VERSION = "8dcb0a8c1c0102c87224308028632cc986a61183"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+class SectionSize(Enum):
+    """
+    Enum for the section size
+    """
+
+    NORMAL = 1
+    UNDERSIZE = 2
+    OVERSIZE = 3
```

Comment:
```suggestion
class SectionSize(IntEnum):
    """
    Enum for the section size
    """

    NORMAL = 0
    UNDERSIZE = 2
    OVERSIZE = 100
```
You can hard code the number values here instead doing the replacement in the function.